settings/Core.setting.php

```diff
@@ -1201,4 +1201,16 @@
     'description' => ts('How long should HTTP requests through Guzzle application run for in seconds'),
     'help_text' => ts('Set the number of seconds http requests should run for before terminating'),
   ],
+  'queue_maintenance_mode' => [
+    'name' => 'queue_maintenance_mode',
+    'type' => 'Boolean',
+    'default' => FALSE,
+    'html_type' => 'yesno',
+    'add' => '5.67',
+    'title' => ts('Is Queue processing in maintenance mode?'),
+    'is_domain' => 1,
+    'is_contact' => 0,
+    'description' => ts('If enabled, CiviCRM will not process background queues.'),
+    'help_text' => ts('This setting will only affect sites that have background queue processing enabled (eg. coworker)'),
+  ],
```

Comment:
@eileenmcnaughton This doesn't seem to actually be used anywhere? Also, does it really need to be called "maintenance mode" when it's actually just a setting to disable queue processing that might be used in various circumstances? I'm guessing it needs to be a setting because it's being read by the external queue processor?
What about something like "queue_disable" and then title "Disable background queue processing".

Reply:
@mattwire argh - it's not being called cos I screwed it up on fixing the listener name ....  I'll fix that. I'm OK with queue_disable

Reply:
@mattwire ok - updated

Reply:
Great! I'm happy with that now

Reply:
cool thanks @mattwire - that also explains why @totten couldn't see my change earlier - I put the listener name in the setting.get call. 

Also - note sure if you saw it but we had a discussion earlier today & notes/recording/transcript are in product maintenance on chat